src/core/server/csp/config.ts

```diff
@@ -18,6 +18,7 @@
  */
 
 import { TypeOf, schema } from '@kbn/config-schema';
+import { IS_KIBANA_DISTRIBUTABLE } from '../../../legacy/utils/artifact_type';
```

Comment:
This is just going to break when we delete this code in a few months. We already have packageInfo available in Core which I think you could pass into the `CspConfig` constructor to and use it to modify the `style-src` rules that actually get consumed.

Reply:
Hmm, not sure I'm following. Would you be opposed to implementing that?

Reply:
You should be able to thread `env.packageInfo` into `new HttpConfig` [here](https://github.com/elastic/kibana/blob/master/src/core/server/http/http_service.ts#L73) and then pass it to `new CspConfig` which can use it to construct the final rules [here](https://github.com/elastic/kibana/blob/master/src/core/server/csp/csp_config.ts#L72)

Reply:
I see that, but the `defaultValue` part of the schema is a part of the module state in https://github.com/elastic/kibana/blob/2598d15ea1b45da641e6fca4f26d86e729429e23/src/core/server/csp/config.ts#L27-L42, how can I influence that with the `CspConfig` constructor?

Reply:
Took a stab at a solution in b92e72e34ffbecfda26b2b64a1763df0f0e3d214, let me know if that's what you had in mind